x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_details_flyout/crawl_details_flyout.tsx

```diff
@@ -29,6 +30,12 @@ export const CrawlDetailsFlyout: React.FC = () => {
   const { closeFlyout, setSelectedTab } = useActions(CrawlDetailLogic);
   const { crawlRequestFromServer, dataLoading, flyoutClosed, selectedTab } =
     useValues(CrawlDetailLogic);
+  const { fetchLogRetention } = useActions(LogRetentionLogic);
+  const { logRetention } = useValues(LogRetentionLogic);
+
+  useEffect(() => {
```

Comment:
This could use code coverage for your updates. You can cherry-pick this commit if that helps: https://github.com/elastic/kibana/commit/7d0074dad3c7faded329ad73d64ecf9a116a81b5

Reply:
Did this for you to keep things moving.